src/AdsConsentPurposes.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+export enum AdsConsentPurposes {
```

Comment:
Did these come from here somewhere? https://github.com/InteractiveAdvertisingBureau/iabtcf-es/tree/master/modules/core

It would be good to have a reference  in the comments of how these came to be as I imagine they will change over time

Reply:
I added links and a very brief description, let me know if that is not what you had in mind.

Reply:
perfect thanks - the added links sprinkled throughout at least give a handle to all the context an interested dev (like myself) needs to go digging.